packages/web-components/src/tabs/tabs.styles.ts

```diff
@@ -0,0 +1,258 @@
+  durationSlow,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontSizeBase400,
+  fontWeightRegular,
+  lineHeightBase300,
+  lineHeightBase400,
+  spacingHorizontalMNudge,
+  spacingHorizontalSNudge,
+  spacingVerticalL,
+  spacingVerticalMNudge,
+  spacingVerticalS,
+  spacingVerticalSNudge,
+  spacingVerticalXXS,
+  strokeWidthThicker,
+} from '../theme/design-tokens.js';
+import { TabTokenNames } from './tabs.js';
+
+export const styles = css`
+  :host {
+    display: grid;
+    box-sizing: border-box;
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    line-height: ${lineHeightBase300};
+    color: ${colorNeutralForeground2};
+    grid-template-columns: auto 1fr auto;
+    grid-template-rows: auto 1fr;
+  }
+
+  :host([disabled]) {
+    cursor: not-allowed;
+    color: ${colorNeutralForegroundDisabled} !important;
+  }
+
+  :host([disabled]) ::slotted(fluent-tab) {
+    pointer-events: none;
+    cursor: not-allowed;
+    color: ${colorNeutralForegroundDisabled} !important;
+  }
+  :host([disabled]) ::slotted(fluent-tab:after) {
+    background-color: ${colorNeutralForegroundDisabled};
+  }
+  :host([disabled]) ::slotted(fluent-tab[aria-selected='true'])::after {
+    background-color: ${colorNeutralForegroundDisabled};
+  }
+  :host([disabled]) ::slotted(fluent-tab:hover):before {
+    content: unset;
+  }
+
+  .tablist {
+    display: grid;
+    grid-template-rows: auto auto;
+    grid-template-columns: auto;
+    position: relative;
+    width: max-content;
+    align-self: end;
+    box-sizing: border-box;
+  }
+  ::slotted([slot='start']),
+  ::slotted([slot='end']) {
+    display: flex;
+    align-self: center;
+  }
+  ::slotted([slot='start']) {
+    margin-inline-end: 11px;
+  }
+  ::slotted([slot='end']) {
+    margin-inline-start: 11px;
+  }
+
+  // TODO: remove active-indicator styling when FAST Pr to remove active indicator is completed.
+  .active-indicator {
+    display: none;
+  }
+
+  .tabpanel {
+    grid-row: 2;
+    grid-column-start: 1;
+    grid-column-end: 4;
+    position: relative;
+  }
+  :host([orientation='vertical']) {
+    grid-template-rows: auto 1fr auto;
+    grid-template-columns: auto 1fr;
+  }
+  :host([orientation='vertical']) .tablist {
+    grid-row-start: 2;
+    grid-row-end: 2;
+    display: grid;
+    grid-template-rows: auto;
+    grid-template-columns: auto 1fr;
+    position: relative;
+    width: max-content;
+    justify-self: end;
+    align-self: flex-start;
+    width: 100%;
+  }
+  :host([orientation='vertical']) .tabpanel {
+    grid-column: 2;
+    grid-row-start: 1;
+    grid-row-end: 4;
+  }
+  :host([orientation='vertical']) ::slotted([slot='end']) {
+    grid-row: 3;
+  }
+
+  :host([appearance='subtle']) ::slotted(fluent-tab:hover) {
+    background-color: ${colorSubtleBackgroundHover};
+    color: ${colorNeutralForeground1Hover};
+  }
+
+  :host([appearance='subtle']) ::slotted(fluent-tab:active) {
+    background-color: ${colorSubtleBackgroundPressed};
```

Comment:
Just so I understand and I called it out before, we have a pseudo element we are targeting in some places and here we are not.